R/importTaxpasta.R

```diff
@@ -61,7 +68,7 @@ importTaxpasta <- function(file) {
         # Create rowData and rowTree
         rowData(tse) <- .create_row_data(biom, ranks)
         .set_ranks_based_on_rowdata(tse, set.ranks = TRUE)
-        tse <- addHierarchyTree(tse)
+	if (add.hierarchy.tree) tse <- addHierarchyTree(tse)
         # Agglomerate to all existing ranks
         tse <- agglomerateByRanks(tse, agglomerate.tree = TRUE)
```

Comment:
agglomerate.tree --> update.tree (Can you change this line)

We updated parameter names this summer, and I believe this imporTaxpasta was merged during the naming process and we did not notice this earlier